### PR TITLE
feat(about): add volunteers page

### PR DESCRIPTION
**Pull request by muamichali**
Continuing work on https://github.com/COVID19Tracking/website/issues/496

### CODE REVIEW

File: src/components/common/volunteers-list.module.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.list {
```
Comment (muamichali) [inline src/components/common/volunteers-list.module.scss:1]: I wasn't sure if we need a unique name or not here.

Comment (kevee) [reply src/components/common/volunteers-list.module.scss:1]: Nope! The classnames in modules get scoped to their modules.

**Comment by muamichali** [comment]
build error

The build process couldn't find _/data/v1/volunteers.json 

Did I miss a step to add it to the archive?

**Comment by kevee** [comment]
The fix you made earlier today is now in Netlify, so we are live at:

https://deploy-preview-647--upbeat-lovelace-3e9fff.netlify.app/about-project/volunteers

**Comment by muamichali** [comment]
- [ ] We have to stack the columns in mobile...

**Comment by sarawb** [comment]
Got a couple logic issues for you! 

- There's duplication between leads and contributors list; we shouldn't show in contributors if they're already in leads. 
- Leads is alphabetized by last name, contributors by first name. No strong opinions about which we use but I'd like both to be ordered by the same thing. Guessing FN all around will be easier, since we don't use separate FN/LN fields in Slack for this. 

(Design review still tk)

**Comment by sarawb** [comment]
Oh, just noticed that links for users who did not enter http:// before their websites in Slack aren't working. Can we automatically append that? Large and changing volunteer base is gonna make this hard otherwise. Examples: Aviva, Bree, Gabe, Marcie (not a complete list).

**Comment by kevee** [comment]
@sarawb - The main contributors list is automatically pulled from Slack, while the leads list is content from Contentful. I think it's cleaner for us to just ask the leads to disable their listing in Slack, then we can manually manage their public presence in Contentful.

**Comment by schwartzadev** [comment]
>     * [ ]  We have to stack the columns in mobile...
Fixed in 763ec76c7b531ae525c0d1c69613085a4dc47545

> Oh, just noticed that links for users who did not enter http:// before their websites in Slack aren't working. Can we automatically append that? Large and changing volunteer base is gonna make this hard otherwise. Examples: Aviva, Bree, Gabe, Marcie (not a complete list).

Fixed in e479eeb6322593e176a719a6803f57753b0a8396

**Comment by schwartzadev** [comment]
> Leads is alphabetized by last name, contributors by first name. No strong opinions about which we use but I'd like both to be ordered by the same thing. Guessing FN all around will be easier, since we don't use separate FN/LN fields in Slack for this.

Great call! Yes, sorting by first name would definitely be easier. I'll look into sorting the leads that way in contentful.

**Comment by kevee** [comment]
We're holding off on publishing this PR for a few days to give people time to sign up in slack.

**Comment by beep** [comment]
Design-wise, would it be possible to disable the bullets on the contributor list, once we move to multiple columns? I think they’re fine in the single column display, but they look a bit busy on the wider layouts. I might also suggest bringing the line-height down a little bit, maybe to 1.3.

**Comment by careeningspace** [comment]
I noticed that I am on the page twice and Prajakta is not listed as a Lead. Does Prajackta need to edit something in her profile?

**Comment by schwartzadev** [comment]
@careeningspace see Kevin's comment:
> I think it's cleaner for us to just ask the leads to disable their listing in Slack, then we can manually manage their public presence in Contentful.

That way the leads won't appear in the main list below. I think that we'll plan to do this before we merge.

Blocking merge :warning:
-----
- [ ] Team leads need to remove their "Yes I want to be on the website" field on their Slack profiles so they no longer appear in the main contributor section

**Comment by schwartzadev** [comment]
> 
> 
> Design-wise, would it be possible to disable the bullets on the contributor list, once we move to multiple columns? I think they’re fine in the single column display, but they look a bit busy on the wider layouts. I might also suggest bringing the line-height down a little bit, maybe to 1.3.

@beep fixed in 7dc5be2 and 6895ebd :tada:

**Comment by beep** [comment]
HECK YEAH @schwartzadev, thank you for doing that!

**Pull request merged**